src/scss/_variables.scss

```diff
@@ -39,7 +28,7 @@ $_o-forms-checkbox-legend-spacing: oTypographySpacingSize(3); // 12px
 $_o-forms-checkbox-horizontal-spacing: 15px;
 $_o-forms-checkbox-vertical-spacing: oTypographySpacingSize(4); // 16px
```

Comment:
possibly unrelated, but as a pedantic comment, it looks like, of the values that are multiples of 4, half of the variables use `oTypographySpacingSize` and the others don't ... ?

Reply:
Erm some aren't based on typography size directly (like the ones which should align to button height, and others are horizontal rather than vertical spacing (but arguably should use typography spacing).